pkg/store/bucket.go

```diff
@@ -42,6 +42,14 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+// Approximately this is the max number of samples that we may have in any given chunk. This is needed
```

Comment:
super minor: Golang comments for vars/fields should starts with var/field name: `maxSamplesPerChunk`

Reply:
Will do, I agree. Missed that part here. Probably because it isn't exported and I thought I could that rule here but I will change it so that it would match the norm :P